backend/src/routes/users.js

```diff
@@ -1,15 +1,92 @@
+const auth = require('../auth');
 const express = require('express');
 const router = express.Router();
 const models = require('../models');
+const db = require('../db');
+const utils = require('../utils');
+const randtoken = require('rand-token');
 
-router.get('/:id', (req, res) => {
-    res.send('returning specific user ' + req.params.id);
-});
+const getUser = async (req, res) => {
+    try {
+        const phoneNumber = req.body.phoneNumber;
```

Comment:
Why is this phone number read from body? It is a GET request, so should probably be from query params?